### PR TITLE
docs(streaming-files): show how to set content-type and content-dispo…

### PR DESCRIPTION
**Pull request by eamonnmcevoy**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?
StreamableFIle example does not show how to set content-type or content-disposition

Issue Number: 2009

## What is the new behavior?
StreamableFIle example improved

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/techniques/streaming-files.md [hunk 1]
```diff
@@ -39,8 +41,13 @@ import { join } from 'path';
 @Controller('file')
 export class FileController {
   @Get()
+  @Response({ passthrough: true }) res,
```
Comment (kamilmysliwiec) [inline content/techniques/streaming-files.md:44]: I think you meant to add this as a method argument?

**Comment by kamilmysliwiec** [comment]
@jmcdo29 can you take a look 🤞 ?

File: content/techniques/streaming-files.md (hunk 1 above)
Comment (eamonnmcevoy) [reply content/techniques/streaming-files.md]: Yes, updated

**Review by jmcdo29 — COMMENTED** [review]
Could we add a mention that `res.set` is only necessary if we're trying to set the content type and file name, and otherwise the type will be `application/octet-stream`

Other than that, looks good to me

**Comment by kamilmysliwiec** [comment]
Perhaps we should split this into 2 examples(?). Basically, instead of updating the existing one, add another example below saying that `res.set` + `@Res({ passthrough })` is necessary when [..]

File: content/techniques/streaming-files.md [hunk 2]
```diff
@@ -45,3 +45,26 @@ export class FileController {
   }
 }
 ```
+
+The default content type is `application/octet-stream`, if you need to customize the response you can use `res.set`.
+
+> info **hint** If the @Response parameter is present, it must have `{ passthrough: true }`.
```
Comment (kamilmysliwiec) [inline content/techniques/streaming-files.md:51]: Build is failing now (we must escape { } chars)

Comment (eamonnmcevoy) [reply content/techniques/streaming-files.md]: Done

File: content/techniques/streaming-files.md [hunk 3]
```diff
@@ -45,3 +45,26 @@ export class FileController {
   }
 }
 ```
+
+The default content type is `application/octet-stream`, if you need to customize the response you can use `res.set`.
```
Comment (kamilmysliwiec) [inline content/techniques/streaming-files.md:49]:
```suggestion
The default content type is `application/octet-stream`, if you need to customize the response you can use the `res.set` method.
```

**Comment by kamilmysliwiec** [comment]
LGTM

**Pull request merged**